index.html

```diff
@@ -69,45 +74,47 @@ <h1>I'm Alusine, glad to meet you!</h1>
         </div>
       </section>
       <!-- End of first headline -->
-      <section class="works bg-white">
+      <section class="works">
         <div class="card">
           <div class="card-image-container card-image"></div>
-          <div class="card-content">
-            <h3>Tonic</h3>
-            <div class="first-links">
-              <a href="#" class="canopy-style">CANOPY</a>
+          <div class="text1">
+            <div class="card-content">
+              <h3>Tonic</h3>
+              <div class="first-links">
+                <a href="#" class="canopy-style">CANOPY</a>
 
-              <span class="canopy-image"></span>
-              <a href="#">Back End dev</a>
+                <span class="canopy-image"></span>
+                <a href="#">Back End Dev</a>
 
-              <span class="back-end-image"></span>
-              <a href="#">2015</a>
-            </div>
-            <p>
-              A daily selection of privately personalized reads; no accounts or
-              sign-ups required.
-            </p>
-            <div class="second-links">
-              <ul>
-                <li><a href="#">html</a></li>
-                <li><a href="#">css</a></li>
-                <li><a href="#">javascript</a></li>
-              </ul>
-            </div>
-            <div class="see-project">
-              <a href="#">See Project</a>
+                <span class="back-end-image"></span>
+                <a href="#">2015</a>
+              </div>
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <div class="second-links">
+                <ul>
+                  <li><a href="#">html</a></li>
+                  <li><a href="#">css</a></li>
+                  <li><a href="#">javascript</a></li>
+                </ul>
+              </div>
+              <div class="see-project">
+                <a href="#">See Project</a>
+              </div>
```

Comment:
- [ ] One of your project buttons seems like it is disabled as you can see in the image below. Kindly fix it so it conforms to the Figma template

![image](https://user-images.githubusercontent.com/90556221/178105564-d1cb1150-a6e5-48a3-88f7-0d479297301f.png)